src/vm/i386/cgenx86.cpp

```diff
@@ -384,7 +384,7 @@ void HelperMethodFrame::UpdateRegDisplay(const PREGDISPLAY pRD)
 #endif // DACCESS_COMPILE
 
     pRD->pCurrentContext->Eip = pRD->ControlPC = m_MachState.GetRetAddr();
-    pRD->pCurrentContext->Esp = pRD->pCurrentContext->ResumeEsp = pRD->SP = (DWORD) m_MachState.esp();
+    pRD->pCurrentContext->Esp = pRD->SP = (DWORD) m_MachState.esp();
```

Comment:
Should we make the same change in TailCallFrame::UpdateRegDisplay and HijackFrame::UpdateRegDisplay? They set the Esp and ResumeEsp the same way too.